docs/guidelines-for-setting-up-a-local-workspace.md

```diff
@@ -45,55 +45,159 @@ Under Ubuntu Linux, you can follow the [documentation from the Ubuntu Community]
-   2. Go to File > Settings > Editor > Code Style, choose a code style (or create a new one) 
-   3. Click on the settings wheel (next to the scheme chooser), then click "Import Scheme" and choose "IntelliJ Code Style xml". Select the IntelliJ configuration file `config/IntelliJ Code Style.xml`. Click OK.
-   4. Go to File -> Settings -> Checkstyle and import the CheckStyle configuration file. Activate it.
-6. Use the provided run configuration: Run -> Run "JabRef Main"
-
-### Set-up Eclipse
-
-1. Run `./gradlew run` to generate all resources and to check if jabref runs. (This step is only required once)
-2. Run `./gradlew eclipse` (This has to be always execute, when there are new upstream changes)
-7. Copy the file Log4jPlugins.java from `build\generated\sources\annotationProcessor\java\main\org\jabref\gui\logging\plugins` to `org.jabref.gui.logging.plugins`
-8. Create a run/debug configuration for main class `org.jabref.JabRefLauncher` 
-9. In the arguments tab enter the same runtime arguments as above for intellij.
-10. Optional: Install the [e(fx)clipse plugin](http://www.eclipse.org/efxclipse/index.html) from the eclipse marketplace
+
+5. Use IntellJ to build and run (instead of gradle):
+   * File -> Settings -> Build, Execution, Deployment -> Build Tools -> Gradle:
+     * Set "Build and run using" and "Run tests using" both to "IntelliJ IDEA"
+6. Ensure that the JDK13 is enabled for Gradle: Use IntellJ to build and run (instead of gradle):
+   * File -> Settings -> Build, Execution, Deployment -> Build Tools -> Gradle -> Gradle:
+     * Set "Gradle JVM" to "JDK13"
+7. Use the provided code style:
+   1. Install the [CheckStyle-IDEA plugin](http://plugins.jetbrains.com/plugin/1065?pr=idea), it can be found via the plug-in repository:
+      1. File > Settings > Plugins > Marketplace
+      2. Search for "Checkstyle" and choose "CheckStyle-IDEA"
+      3. Close the settings afterwards and restart IntelliJ
+   2. Go to File > Settings > Editor > Code Style:
+      1. Click on the settings wheel (next to the scheme chooser), then click "Import Scheme" and choose "IntelliJ Code Style.xml".
+      2. Select the IntelliJ configuration file `config/IntelliJ Code Style.xml`.
+      3. Click "OK".
+   4. Go to File -> Settings -> Checkstyle -> Configuration File:
+      1. Import the CheckStyle configuration file by clicking the [+] button
+      2. For the description provide e.g. "CheckStyle"
+      3. Click "Browse" and choose `config/checkstyle/checkstyle.xml`
+      4. Click "Next" and "Finish"
+      5. Activate the CheckStyle configuration file by ticking it in the list
+      6. Save settings by clicking "OK"
+8. Open Run -> Edit Configurations...:
+   1. Select Application -> JabRef Main
```

Comment:
Can the two lines here be merged?

Reply:
Do you mean the two lines
```
8. Open Run -> Edit Configurations...:
   1. Select Application -> JabRef Main
```
?
They could. It depends on how one likes it more.

Reply:
Well, you did it in the other breadcrumbs... 😅

Reply:
updated